docs/UserGuide.md

```diff
@@ -936,145 +1052,58 @@ welcome to update data directly by editing that data file.
-    </tr>
-    <tr>
-        <td><code>search</code></td>
-        <td><code>[tagKeyword]</code></td>
-    </tr>
-</table>
+| Command Word    | Action Word | Parameters                                                       |
+|-----------------|-------------|------------------------------------------------------------------|
+| `localcourse`   | `list`      | None.                                                            |
+|                 | `add`       | `[localcode] [localname] [units] [description]`                  |
+|                 | `delete`    | `[localcode]`                                                    |
+|                 | `update`    | `[localcode] [attribute] [newvalue]`                             |
+|                 | `search`    | `[attribute] [query]`                                            |
+|                 | `sort`      | `[attribute]`                                                    |
+| `partnercourse` | `list`      | None.                                                            |
+|                 | `add`       | `[university] [partnercode] [partnername] [units] [description]` |
+|                 | `delete`    | `[university] [partnercode]`                                     |
+|                 | `update`    | `[university] [partnercode] [attribute] [newvalue]`              |
+|                 | `search`    | `[attribute] [query]`                                            |
+|                 | `sort`      | `[attribute]`                                                    |
+| `mapping`       | `list`      | None.                                                            |
+|                 | `add`       | `[localcode] [university] [partnercode] [information]`           |
+|                 | `delete`    | `[localcode] [university] [partnercode]`                         |
+|                 | `search`    | `[attribute] [query]`                                            |
+|                 | `sort`      | `[attribute]`                                                    |
+| `university`    | `list`      | None.                                                            |
+|                 | `search`    | `[query]`                                                        |
+|                 | `sort`      | None.                                                            |
+| `note`          | `list`      | None.                                                            |
+|                 | `add`       | `[content] [tag]`                                                |
+|                 | `delete`    | `[index]`                                                        |
+|                 | `update`    | `[index] [content]`                                              |
+|                 | `tag`       | `[index] [tag]`                                                  |
+|                 | `cleartag`  | `[index]`                                                        |
```

Comment:
Is this necessary? Personally I implemented the rowspan to allow multi-row. Here is a comparison:

![image](https://github.com/AY2324S1-CS2103T-W10-2/tp/assets/121750834/5187eac5-255f-4621-9bfc-2465b819a985)

I would suggest that we leave the current implementation be, as it accurately reflects contribution:
![image](https://github.com/AY2324S1-CS2103T-W10-2/tp/assets/121750834/0582bbdd-89f7-486d-b941-4bb3cecef28e)

Reply:
This part of the code is not meant to be merged. This pull request was still a draft at the point of your review.

Reply:
Thanks for your clarification on this.